test/unit/TestInstrumentation.cpp

```diff
@@ -7,11 +7,11 @@
 #include <thread>
 
 int main(int argc, char *argv[]) {
-  const char opstart[] = "TOpStar";
-  const char op2[]     = "TOp2";
-  const char op3[]     = "TOp3";
-  const char op4[]     = "TOp4";
-  const char opfinal[] = "TOpFin";
+  const char opstart[8] = "TOpStar";
+  const char op2[8]     = "TOp2";
```

Comment:
Should this be 5 instead of 8? The same for the following ones.

Reply:
It is 8 to make sure the allocation is 64-bits to remove the GCC warning in #1230.

Reply:
Which warnings did you get if using `const char op2[5]     = "TOp2";`? It is clear that the string has 5 chars including the null terminator.

Reply:
`TestInstrumentation.cpp:19:20: warning: array subscript ‘const int64_t {aka const long int}[0]’ is partly outside array bounds of ‘const char [5]’ [-Warray-bounds]
`
I don't think GCC likes the cast from an array smaller than 8 bytes a pointer to an 8-byte type.

Reply:
It looks to me `char[] ` or `char[5]` is not the root cause. It is the problem of casting to int64_t at Line 19. Changing to 8 is just make GCC do not complain about array bounds but the last 3 bytes are undefined.

Reply:
But there is no problem with undefined bytes at the end of a C string, GCC just seems to care about that the full width of the pointer type used is within bounds. I wonder if this is related to https://gcc.gnu.org/bugzilla/show_bug.cgi?id=98266 or something similar, but I am in no way a GCC expert and regardless changing the test case to avoid the warning seems pretty reasonable. We can't avoid casting, unless we changed the signature of OMInstrumentPoint.